### PR TITLE
Run RuboCop on Travis CI

### PR DESCRIPTION
**Pull request by walro**
Mainly to get something more real than the current 'echo'.

### CODE REVIEW

**Comment by walro** [comment]
@Chrizpy FYI, not sure if you had changed anything that got lost in 7185422. I matched against the [default in 0.90](https://github.com/rubocop-hq/rubocop/blob/v0.90.0/config/default.yml) and that showed a few changes:

```diff
diff --git a/.rubocop.yml b/.rubocop.yml
index 74b95d2..71e4a20 100644
--- a/.rubocop.yml
+++ b/.rubocop.yml
@@ -106,7 +106,7 @@ AllCops:
   # the `--enable-pending-cops` command-line option.
   # When `NewCops` is `disable`, pending cops are disabled in bulk. Can be overridden by
   # the `--disable-pending-cops` command-line option.
-  NewCops: enable
+  NewCops: pending
   # Enables the result cache if `true`. Can be overridden by the `--cache` command
   # line option.
   UseCache: true
@@ -2695,7 +2695,7 @@ Style/ClassVars:
 Style/CollectionMethods:
   Description: 'Preferred collection methods.'
   StyleGuide: '#map-find-select-reduce-include-size'
-  Enabled: true
+  Enabled: false
   VersionAdded: '0.9'
   VersionChanged: '0.27'
   Safe: false
```

Either way, the 👮‍♂️ seems satisfied.

**Comment by Chrizpy** [comment]
> @Chrizpy FYI, not sure if you had changed anything that got lost in [7185422](https://github.com/Chrizpy/Snake-Ruby/commit/718542222872707512414cd93793641af1d6c331). I matched against the [default in 0.90](https://github.com/rubocop-hq/rubocop/blob/v0.90.0/config/default.yml) and that showed a few changes:
> 
> ```diff
> diff --git a/.rubocop.yml b/.rubocop.yml
> index 74b95d2..71e4a20 100644
> --- a/.rubocop.yml
> +++ b/.rubocop.yml
> @@ -106,7 +106,7 @@ AllCops:
>    # the `--enable-pending-cops` command-line option.
>    # When `NewCops` is `disable`, pending cops are disabled in bulk. Can be overridden by
>    # the `--disable-pending-cops` command-line option.
> -  NewCops: enable
> +  NewCops: pending
>    # Enables the result cache if `true`. Can be overridden by the `--cache` command
>    # line option.
>    UseCache: true
> @@ -2695,7 +2695,7 @@ Style/ClassVars:
>  Style/CollectionMethods:
>    Description: 'Preferred collection methods.'
>    StyleGuide: '#map-find-select-reduce-include-size'
> -  Enabled: true
> +  Enabled: false
>    VersionAdded: '0.9'
>    VersionChanged: '0.27'
>    Safe: false
> ```
> 
> Either way, the 👮‍♂️ seems satisfied.

Hmmm I don't remember specifically what I did, but I'm pretty sure I did something... but if 👮‍♂️ is happy, I'm happy.

File: Gemfile.lock [hunk 1]
```diff
@@ -1,13 +1,34 @@
...
+      rainbow (>= 2.2.2, < 4.0)
+      regexp_parser (>= 1.7)
+      rexml
+      rubocop-ast (>= 0.3.0, < 1.0)
+      ruby-progressbar (~> 1.7)
+      unicode-display_width (>= 1.4.0, < 2.0)
+    rubocop-ast (0.3.0)
+      parser (>= 2.7.1.4)
+    ruby-progressbar (1.10.1)
     ruby2d (0.9.4)
```
Comment (Chrizpy) [inline Gemfile.lock:23]: Is this default settings required by Travis?

**Comment by walro** [comment]
I take it it's OK to merge.

File: Gemfile.lock (hunk 1 above)
Comment (Chrizpy) [reply Gemfile.lock:23]: Sorry I forgot to submit the review, still a bit new to GitHub's features :P

Comment (walro) [reply Gemfile.lock:23]: > Is this default settings required by Travis?

You mean the gems added to the lock file? Yes, these are all the dependencies needed to get RuboCop going.

```
$ gem dependency rubocop
Gem rubocop-0.90.0
  bundler (>= 1.15.0, < 3.0, development)
  parallel (~> 1.10)
  parser (>= 2.7.1.1)
  rainbow (>= 2.2.2, < 4.0)
  regexp_parser (>= 1.7)
  rexml (>= 0)
  rubocop-ast (>= 0.3.0, < 1.0)
  ruby-progressbar (~> 1.7)
  unicode-display_width (>= 1.4.0, < 2.0)

Gem rubocop-ast-0.3.0
  bundler (>= 1.15.0, < 3.0, development)
  parser (>= 2.7.1.4)
```

An alternative, if you don't want to add all these dependencies to your project, is to do something like the following on Travis CI:

```yml
before_script:
  - gem install rubocop -v 0.90.0
script:
  - rubocop
```

But, IMHO, it's nice to `rubocop` in the `Gemfile` as you will then be certain everyone contributing to the project is running the same version locally as on Travis CI.

Comment (Chrizpy) [reply Gemfile.lock:23]: Alright I see what you mean, sweet!

**Pull request merged**